src/lesson6/classwork/table_ddl.sql

```diff
@@ -1,27 +1,27 @@
 CREATE TABLE student
 (
-    student_id serial,
-    first_name varchar,
-    last_name varchar,
-    birthday date,
-    phone varchar
+    student_id BIGSERIAL,
```

Comment:
Teacher has made mistake. You don't have to write "student_id", because this column is already located in table student, so it would better to write just "id".

Exemple:
CREATE TABLE student
(
    id BIGSERIAL,
    first_name VARCHAR,
    last_name VARCHAR,
    birthday DATA,
    phone VARCHAR
);

Reply:
Please, make sure everywhere is named as "id".